.github/workflows/ci.yml

```diff
@@ -70,8 +68,18 @@ jobs:
                         </settings>' > ~/.m2/settings.xml
             - name: print file
               run: cat ~/.m2/settings.xml
-            - name: Maven Build
-              run: mvn clean install 
+            # Grant execute permission to the gradlew script
+            - name: Grant execute permission for gradlew
+              run: chmod +x gradlew
+
+            # Build the project with Gradle
+            - name: Build with Gradle
+              env:
+                packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+                packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+                JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
+              run: |
+                ./gradlew build
             - name: Ballerina Download
```

Comment:
You can use the `setup-ballerina` github action instead of downloding manually. Refer to https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/blob/14cec8673204743ad714fbdf9cca877470a17633/.github/workflows/build-with-bal-test-native.yml#L46

Reply:
lets apply this to the other workflows as well.